addons/editor/initKeybinds.sqf

```diff
@@ -82,3 +82,84 @@
         };
     };
 }, {}, [0, [false, false, false]]] call CBA_fnc_addKeybind; // Default: Unbound
+
+[ELSTRING(main,DisplayName), QGVAR(watchCuratorCamera), [LSTRING(WatchCuratorCamera), LSTRING(WatchCuratorCamera_Description)], {
+    if (!isNull curatorCamera && {!GETMVAR(RscDisplayCurator_search,false) && {count SELECTED_OBJECTS > 0}}) then {
+        private _gunners = (SELECTED_OBJECTS apply {gunner vehicle _x}) - [objNull];
+        private _pos = getPos curatorCamera;
+        {
+            [QEGVAR(common,doWatch), [_x, _pos], _x] call CBA_fnc_targetEvent;
+        } forEach _gunners;
+
+        true // handled
+    };
+}, {}, [0, [false, false, false]]] call CBA_fnc_addKeybind; // Default: Unbound
+
+[ELSTRING(main,DisplayName), QGVAR(watchCursor), [LSTRING(WatchCursor), LSTRING(WatchCursor_Description)], {
+    if (!isNull curatorCamera && {!GETMVAR(RscDisplayCurator_search,false) && {count SELECTED_OBJECTS > 0}}) then {
+        curatorMouseOver params ["_type", "_entity", ""];
+        private _isCancelling = _type == "OBJECT" && {_entity in SELECTED_OBJECTS};
+        private _gunners = (SELECTED_OBJECTS apply {gunner vehicle _x}) - [objNull];
+
+        private _cursorPosASL = [] call EFUNC(common,getPosFromScreen);
+        {
+            [QEGVAR(common,doWatch), [_x, [ASLToAGL _cursorPosASL, objNull] select _isCancelling], _x] call CBA_fnc_targetEvent;
+        } forEach _gunners;
+
+        true // handled
+    };
+}, {}, [0, [false, false, false]]] call CBA_fnc_addKeybind; // Default: Unbound
+
+[ELSTRING(main,DisplayName), QGVAR(forceFire), [LSTRING(ForceFire), LSTRING(ForceFire_Description)], {
+    if (!isNull curatorCamera && {!GETMVAR(RscDisplayCurator_search,false) && {count SELECTED_OBJECTS > 0}}) then {
+        private _shooters = SELECTED_OBJECTS select {!isNull group _x};
+        [QEGVAR(common,ForceFire), [clientOwner, _shooters], _shooters] call CBA_fnc_targetEvent;
+
+        true // handled
+    };
+}, {
+    [QEGVAR(common,ForceFire), [clientOwner, []]] call CBA_fnc_localEvent;
+    [QEGVAR(common,ForceFire), [clientOwner, []]] call CBA_fnc_remoteEvent;
```

Comment:
```suggestion
    [QEGVAR(common,ForceFire), [clientOwner, []]] call CBA_fnc_globalEvent;
```
Should have the same effect, shouldn't it?

Reply:
Yes! idk why I couldn't find that one.